pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -358,6 +335,7 @@ func (ctrl *csiSnapshotSideCarController) deleteCSISnapshotOperation(content *cr
 
 	_, snapshotterCredentials, err := ctrl.getCSISnapshotInput(content)
 	if err != nil {
+		ctrl.eventRecorder.Event(content, v1.EventTypeWarning, "SnapshotDeleteError", "Failed to get snapshot class")
```

Comment:
getCSISnapshotInput could fail other than cannot get snapshot class.

s/Failed to get snapshot class/Failed to get snapshot deletion input.